src/middleware/validator/user-course.validator.ts

```diff
@@ -0,0 +1,18 @@
+import { check } from 'express-validator'
+
+import { userCourseRoles } from 'devu-shared-modules'
+
+import validate from './generic.validator'
+
+const userId = check('userId').isNumeric()
+const courseId = check('courseId').isNumeric()
+const level = check('level')
+  .isString()
```

Comment:
probably don't need the `isString` here since it'll match exactly to what's in the `userCourseRoles`